docs/release-notes/v12.2.1.md

```diff
@@ -0,0 +1,100 @@
+---
+id: v12.2.1
+title: Version 12.2.1 Release Notes
+sidebar_label: v12.2.1
+image: "img/release-note-link-preview.png"
+---
+
+_February 15th, 2024_
+
+
+:::info
+Mia-Platform Console v12.2.1 is **now in Preview!**
+
+Console SaaS users can try out v12.2.1 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## Console
+
+## Fast Data
+
+### Fast Data Services
+
+#### Single View Trigger Generator
+
+The new version `v3.1.5` of the _Single View Creator_ is available!
+
+##### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* in case of one-to-many relationships, the service does not crash anymore when it executes an automatic strategy over a record whose primary key fields have a value set to `null`
+
+For more information check out the service [changelog](/runtime_suite/single-view-trigger-generator/changelog.md).
+
+#### Single View Creator
+
+In order to simplify the creation of Single View Creator plugins depending on the selected Fast Data Architecture, the original marketplace plugin has been split into two. The Mongo based plugin is  preconfigured for the [Standard architecture](/fast_data/architecture.md#standard-architecture), while the Kafka based is preconfigured for the [Event-Driven architecture](/fast_data/architecture.md#event-driven-architecture).
+
+![Single View Creator Marketplace](./img/v12.2.1/svc-marketplace.png)
+
+## Data Catalog
+
+#### Data Catalog Agent Template
+
+The new version `v0.4.0` of the _Data Catalog Agent_ template is available!
+
+##### New Features
+
+- Mia-Platform CRUD Service (`v6.10.0` or later) has been added among the supported Data Sources, so that it is possible to configure the Agent to gather metadata from the ones deployed in your projects environments.
```

Comment:
```suggestion
Mia-Platform CRUD Service (`v6.10.0` or later) has been added among the supported Data Sources, so that it is possible to configure the Agent to gather metadata from the ones deployed in your projects environments.
```